examples/multi-language/README.md

```diff
@@ -28,6 +28,8 @@ This project provides examples of Apache Beam
 * **python/javacount** - A Python pipeline that counts words using the Java `Count.perElement()` transform.
 * **python/javadatagenerator** - A Python pipeline that produces a set of strings generated from Java.
                                   This example demonstrates the `JavaExternalTransform` API.
+* **python/wordcount_external** - A Python pipeline that runs the Word Count workflow using three external Java
+                transforms. This example demonstrates the simpler `ExternalTransformProvider` API.
```

Comment:
Do we have any step-by-step guide somewhere about how to create a new JavaExternalTransform? If so, can we link it here? When I first looked at the java codes, I am a bit lost about what parts I need to create in order to use `ExternalTransformProvider`.

Reply:
Clarifying that we are using SchemaTransforms here, not JavaExternalTransform.
Also modifying comments in wordcount_external.py to clarify which Java transforms are used for this example